api/account/deduplication/lifo.py

```diff
@@ -9,31 +9,31 @@
 
 
 # --> LIFO deduplication
-def lifo(
-    community: Community, lifo_passport: dict, address: str
-) -> Tuple[dict, list | None]:
-    deduped_passport = copy.deepcopy(lifo_passport)
-    deduped_passport["stamps"] = []
-    if "stamps" in lifo_passport:
-        for stamp in lifo_passport["stamps"]:
-            stamp_hash = stamp["credential"]["credentialSubject"]["hash"]
-
-            # query db to see if hash already exists, if so remove stamp from passport
-            if (
-                not Stamp.objects.filter(hash=stamp_hash, passport__community=community)
-                .exclude(passport__address=address)
-                .exists()
-            ):
-                deduped_passport["stamps"].append(copy.deepcopy(stamp))
-
-    return (deduped_passport, None)
+# TODO: remove this function as it is not used
+# def lifo(
+#     community: Community, lifo_passport: dict, address: str
+# ) -> Tuple[dict, list | None]:
+#     deduped_passport = copy.deepcopy(lifo_passport)
+#     deduped_passport["stamps"] = []
+#     if "stamps" in lifo_passport:
+#         for stamp in lifo_passport["stamps"]:
+#             stamp_hash = stamp["credential"]["credentialSubject"]["hash"]
+
+#             # query db to see if hash already exists, if so remove stamp from passport
+#             if (
+#                 not Stamp.objects.filter(hash=stamp_hash, passport__community=community)
+#                 .exclude(passport__address=address)
+#                 .exists()
+#             ):
+#                 deduped_passport["stamps"].append(copy.deepcopy(stamp))
+
+#     return (deduped_passport, None)
```

Comment:
I would prefer if we just go ahead and delete this, but no big deal!

Reply:
Done